frontend/add-client/confirm.component.tsx

```diff
@@ -102,7 +102,7 @@ export default function Confirm(props: StepComponentProps) {
     employmentSector: c.employmentSector,
     payInterval: c.payInterval,
     weeklyEmployedHours: c.weeklyEmployedHours,
-    householdIncome: c.householdIncome,
+    householdIncome: c.householdIncome || null,
```

Comment:
Why is this here?

Reply:
Does it not need to be to make it nullable?

Reply:
I'm just confused why it's in this PR. The pr is for api docs, but this is inside of the frontend code.

Reply:
Shoot your right. I missed a git stash doing this one. That should only be in the other PR. Sorry about that.